modules/ROOT/pages/tokenization-example.adoc

```diff
@@ -167,6 +138,7 @@ image::tokenization-example-add-tls-enabled-gateway.png[]
 . Click *Deploy*. +
 Once the application has a status of "Running" you are ready to test.
 
+//// 
```

Comment:
Do you have a JIRA to add this back or is it hidden debt?

Reply:
None of this is hidden debt. It's a soft delete since the content is still in progress and some of it will be valid in a future iteration.